api/response/schema_query.py

```diff
@@ -0,0 +1,75 @@
+import graphene
+from api.response.models import Response as ResponseModel
+from graphql import GraphQLError
+from api.room.schema import Room
+from api.missing_items.models import MissingItems as MissingItemsModel
+from api.room_resource.models import Resource
+from helpers.auth.authentication import Auth
+
+
+class ResponseDetails(graphene.ObjectType):
```

Comment:
@Bonifase We have discussed in our sync why it is necessary to go by this method, can you kindly approve now if there is no other concern

Reply:
@cwizard2011 could you summarise your discussion in this thread as well for other reviewers to follow along?

Reply:
Thanks, @joshuaocero, I have two classes because when a user reports multiple missing items, we are going to have more than one missing item in the table with the same response id because they all come from a single response, so to have a unique response with multiple missing item in form of a list, I created a class to handle the response details for each unique response ID, and another class which has a response field and takes a list inherited from the first class